### DISCUSSION TITLE
HELP WITH NVAPI - I WILL PAY YOU

### DISCUSSION THREAD

**Discussion by silvabizti** [Rem0o/FanControl.Releases#2332] [General]
I can't set up the program because of this error. Can anyone help me solve it? I will pay you on paypal if you do. My pc is a omen hp 15 and it has been going crazy with the fan noise lately. I need to tame it or I wont be able to use it at work
![Sem título](https://github.com/Rem0o/FanControl.Releases/assets/161554394/21ad4e40-64f9-464d-a10a-c3634d5c5fce)

**Comment by Rem0o** [comment]
That's a laptop? Laptop fans are not hooked to NvAPI. Only desktop cards are. Sorry.

**Reply by silvabizti** [reply]
Yes, it is. So there is no way to use the program with a laptop?

**Reply by Rem0o** [reply]
Not as of now.
